src/common/components/signUpModal/signUpModal.component.js

```diff
@@ -36,6 +36,9 @@ class SignUpModalController {
         this.onSuccess(data)
       }, (error) => {
         this.hasError = true
+        if (error?.data?.error === 'invalid_password' && error?.data?.description) {
```

Comment:
Note this whole commit will be no longer useful when we move to Okta.

Reply:
I see in this UI Engineering OKR Tactics & Resource Allocation spreadsheet we have an OKR to make it easy for staff members to log in via OKTA, but what about non-staff members? Will they also use OKTA?

Reply:
Yep, everyone will use Okta when we make the switch, which will get rid of our custom modal taking username/password and instead redirect the user directly to Okta's sign in form.